docs/framework/develop-web-apps-with-aspnet.md

```diff
@@ -20,7 +20,7 @@ ms.author: "tdykstra"
 manager: "wpickett"
 ---
 # Developing Web Applications with ASP.NET
-ASP.NET is a .NET Framework technology for creating web apps. Documentation for ASP.NET is in the [Web Development](http://go.microsoft.com/fwlink/p/?LinkID=235794) section of the MSDN Library and on the [www.asp.net website](http://go.microsoft.com/fwlink/p/?LinkId=251042). The following links are provided for your convenience:  
+ASP.NET is a .NET Framework technology for creating web apps. Documentation for ASP.NET is in the [Web Development](http://go.microsoft.com/fwlink/p/?LinkID=235794) section of Microsoft Docs and on the [www.asp.net website](https://www.asp.net). The following links are provided for your convenience:  
```

Comment:
Let's cut this entire sentence: Documentation for ASP.NET is in the [Web Development](http://go.microsoft.com/fwlink/p/?LinkID=235794) section of Microsoft Docs and on the [www.asp.net website](https://www.asp.net).